myuw/static/vue/components/classlist/class-student-list.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <uw-card v-if="showCard"
+           :loaded="isReady"
+           :errored="isErrored"
+           :errored-show="isErrored"
```

Comment:
This is not required as errored-show is true by default. The error will not be shown is errored is false.

Reply:
Thanks, updated!